lib/Controller/Layout.php

```diff
@@ -272,6 +272,8 @@ public function displayDesigner(Request $request, Response $response, $id)
             'users' => $this->userFactory->query(),
             'modules' => $moduleFactory->getAssignableModules(),
             'timeZones' => $timeZones,
+            'help' => $this->getHelp()->link('Layout', 'Edit'),
+
```

Comment:
phpcs

Reply:
the empty new line between the new code and the end of the array right?